src/features/hotspots/details/HotspotDetailsHandle.tsx

```diff
@@ -1,28 +1,41 @@
-import { Hotspot } from '@helium/http'
-import React, { memo } from 'react'
+import React, { memo, useCallback } from 'react'
+import Chevron from '@assets/images/chevron-right.svg'
+import { StyleSheet } from 'react-native'
 import Box from '../../../components/Box'
-import CardHandle from '../../../components/CardHandle'
-import HotspotMoreMenuButton from './HotspotMoreMenuButton'
-import FollowButton from '../../../components/FollowButton'
+import TouchableOpacityBox from '../../../components/TouchableOpacityBox'
+import { useAppDispatch } from '../../../store/store'
+import hotspotsSlice from '../../../store/hotspots/hotspotsSlice'
+
+const HotspotDetailsHandle = () => {
+  const dispatch = useAppDispatch()
+
+  const prev = useCallback(() => {
+    dispatch(hotspotsSlice.actions.selectPrevHotspot())
+  }, [dispatch])
+
+  const next = useCallback(() => {
+    dispatch(hotspotsSlice.actions.selectNextHotspot())
```

Comment:
If you select the followed hotspots filter when you have no followed hotspots and then go to the map and select a hotspot and tap the next arrow, you end up on a blank white card

Reply:
Also if you are following only one hotspot or I'm guessing have only one hotspot, the next and previous don't do anything. Maybe we should hide them?